src/utils/aquarius.ts

```diff
@@ -35,7 +35,7 @@ export const MAXIMUM_NUMBER_OF_PAGES_WITH_RESULTS = 476
 export function getFilterTerm(
   filterField: string,
   value: string | number | boolean | number[] | string[],
-  key: 'terms' | 'term' | 'match' = 'term'
+  key: 'terms' | 'term' | 'match' | 'match_phrase' = 'term'
 ): FilterTerm {
   const isArray = Array.isArray(value)
   const useKey = key === 'term' ? (isArray ? 'terms' : 'term') : key
```

Comment:
nitpick(non-blocking): This looks like 'clever' code. There might be an easier way to express this in more than one line.

Reply:
You mean by splitting the options from the default value assignation?

Reply:
I thought about this, too: A dedicated type might be useful now.
```ts
type TFilterType = 'terms' | 'term' | 'match' | 'match_phrase'
type TFilterValue = string | number | boolean | number[] | string[]

export function getFilterTerm(
  field: string,
  value: TFilterValue,
  type: 'terms' | 'term' | 'match' = 'term'
) : FilterTerm {}
```